ci/aws-analytical-env/jobs/qa.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - get: emr-launcher-release
         trigger: true
       - get: manage-mysql-user-release
-        trigger: false
+        trigger: true
```

Comment:
I guess setting this to true doesnt have any adverse effects, I think we only the mysql stuff to rotate passwords which are always user triggered.

Reply:
Yeah it should trigger and if we want to pin it we can it in the pipeline.